.github/workflows/vib-early-feedback.yaml

```
@@ -63,25 +64,28 @@ jobs:
       # Source: https://github.community/t/no-output-on-process-completed-with-exit-code-1/123821/3
       - id: show-error
         name: 'Show error'
-        if: ${{steps.get-chart.outputs.result == 'fail'}}
+        if: ${{ steps.get-chart.outputs.result == 'fail' }}
         uses: actions/github-script@v3
         with:
           script: |
-              core.setFailed('${{steps.get-chart.outputs.error}}')
+              core.setFailed('${{ steps.get-chart.outputs.error }}')
   vib-execution:
     runs-on: ubuntu-latest
     needs: get-chart
-    if: ${{needs.get-chart.outputs.result == 'ok'}}
-    name: 'Package, Lint and Trivy scan'
+    if: ${{ needs.get-chart.outputs.result == 'ok' }}
+    name: 'Package, Lint, Trivy scan & Deployment'
     steps:
       - uses: actions/checkout@v2
         name: 'Checkout Repository'
         with:
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
       - uses: vmware-labs/vmware-image-builder-action@main
-        name: 'Package, Lint and Trivy scan'
+        name: 'Package, Lint, Trivy scan & Deployment'
```

```suggestion
        name: 'Package, Lint, Trivy scan & Health Check'
```
WDYT? Health Check contains a deployment and it might be more understandable to internal/external readers.

---

I will close this, it does no longer apply.